build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.0.0-alpha1'
```

Comment:
This is a beta/alpha release of software. Like all software that is in a pre-release state there could be bugs that haven't been addressed yet. See the [release announcement](http://android-developers.blogspot.com/2015/11/android-studio-20-preview.html) for more info.

The purpose of these sample repositories are to give developers example code, code style, and patterns to use in their production applications. It is VERY bad practice to put in pre-release dependencies into your production applications.